Library/Formula/uwsgi.rb

```diff
@@ -45,6 +45,10 @@ class Uwsgi < Formula
   depends_on "v8" => :optional
   depends_on "zeromq" => :optional
 
+  # Fix build on case-sensitive filesystems
+  # https://github.com/Homebrew/homebrew/issues/45560
+  patch :DATA
```

Comment:
Is it worth doing this as an `inreplace` rather than a `patch` if it's just one line to change?